### DISCUSSION TITLE
Skia rendering issues in Windows 10 and Windows 11

### DISCUSSION THREAD

**Discussion by lemalcs** [unoplatform/uno#17411] [Q&A] [answered]
Hello, I have a Skia project built with Uno Platform but I'm finding rendering issues when running it on some Windows versions.

When you click outside the dark rectangle, it disappears, but shows up again, when you focus on the text boxes.

The expected behavior is that the **whole window background** is light or dark, depending on the current OS theme.

On Windows 10 x64 Version 1507 (RTM) (OS build 10240)
![image](https://github.com/unoplatform/uno/assets/48371435/2e8ba173-3799-4502-838e-06040c5c0a58)

On Windows 11 Pro (x64) Version 23H2 build 22631.3593:
![image](https://github.com/unoplatform/uno/assets/48371435/a61a0971-56ff-4a76-978c-35d6177e1893)

Interestingly on Windows 7 SP1 x64, it is rendered with any issues:
![image](https://github.com/unoplatform/uno/assets/48371435/d7e8e262-c09d-4083-83d4-21da579ab460)

This is the command I used to publish the project (.NET desktop runtime 8 dependant):

`dotnet publish -f net8.0-desktop -c Release -o ./publish`

## My development environment
- Uno Platform 5.2.56.70
- Visual Studio 2022 (64 bits) Version 17.10.3
- Windows 11 Pro (x64) Version 23H2 build 22631.3737

dotnet --list-sdks:

```
5.0.416 [C:\Program Files\dotnet\sdk]
7.0.120 [C:\Program Files\dotnet\sdk]
8.0.106 [C:\Program Files\dotnet\sdk]
8.0.206 [C:\Program Files\dotnet\sdk]
8.0.300 [C:\Program Files\dotnet\sdk]
8.0.302 [C:\Program Files\dotnet\sdk]
9.0.100-preview.5.24307.3 [C:\Program Files\dotnet\sdk]
```

When I run the same application in my development environment, it renders as expected:

With dark theme
![image](https://github.com/unoplatform/uno/assets/48371435/fb3e93cb-7bdc-422f-87a2-a9d9f021cba2)

With light theme
![image](https://github.com/unoplatform/uno/assets/48371435/2e3802af-5df8-4a56-b421-b8c084e28c23)

Here is the [sample project](https://github.com/lemalcs/UnoAppSkiaDesktop) I used for the tests; it's an almost fresh project I just added two text boxes, two labels, and one button.

Is there any workaround for this problem or something I'm missing around? 

I appreciate any help you can provide. 

Thanks in advance.

**Comment by ramezgerges** [comment]
Hey @lemalcs, your sample repo is private.

**Comment by lemalcs** [comment]
I'm sorry, the repo is public now. Thanks

**Comment by ramezgerges** [comment]
I couldn't reproduce this. @Youssef1313 @MartinZikmund could you also try it out? It seems very fragile to reproduce, so maybe you'll have better luck than me.

**Comment by MartinZikmund** [comment, marked as answer]
Converted to an issue https://github.com/unoplatform/uno/issues/17526, let's continue investigating there
